ecosystem/typescript/sdk/examples/typescript/multisig_account.ts

```diff
@@ -83,12 +83,6 @@ const { AccountAddress, EntryFunction, MultiSig, MultiSigTransactionPayload, Tra
   const multisigTxExecution = new TransactionPayloadMultisig(
     new MultiSig(AccountAddress.fromHex(multisigAddress), transferTxPayload),
   );
-  // We can simulate the transaction to see if it will succeed without having to create it on chain.
-  const [simulationResp] = await client.simulateTransaction(
-    owner2,
-    await client.generateRawTransaction(owner2.address(), multisigTxExecution),
-  );
-  assert(simulationResp.success);
```

Comment:
We don't want to keep this?

Reply:
This is the old way to simulate a multisig transaction before creation. After this PR, one should simulate multisig simulations using multisig accounts as the sender without the auth key check. I will update the TS examples accordingly once the SDK supports `NoAccountSignature`.